### PR TITLE
loading screen

### PR DESCRIPTION
**Pull request by michaelkirk**
I didn't spend too much time on this, but I think it will help assure people that something is happening. Currently it kind of feels broken.

loading_screen mov

### CODE REVIEW

File: game/index.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
...ale(0.5, 0.5)}}@-moz-keyframes ball-atom-size{50%{-moz-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@-o-keyframes ball-atom-size{50%{-o-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@keyframes ball-atom-size{50%{-webkit-transform:scale(0.5, 0.5);-moz-transform:scale(0.5, 0.5);-o-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@-webkit-keyframes ball-atom-zindex{50%{z-index:10}}@-moz-keyframes ball-atom-zindex{50%{z-index:10}}@-o-keyframes ball-atom-zindex{50%{z-index:10}}@keyframes ball-atom-zindex{50%{z-index:10}}@-webkit-keyframes ball-atom-shrink{50%{-webkit-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@-moz-keyframes ball-atom-shrink{50%{-moz-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@-o-keyframes ball-atom-shrink{50%{-o-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@keyframes ball-atom-shrink{50%{-webkit-transform:translate(-50%, -50%) scale(0.8, 0.8);-moz-transform:translate(-50%, -50%) scale(0.8, 0.8);-o-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}
```
Comment (michaelkirk) [inline game/index.html:10]: I had to try a few different animations - it was fun finding out which things work while the main thread is blocked.

**Comment by michaelkirk** [comment]
I'm not sure where your osm demo lives, but if you like it, you could include it there too.

**Comment by michaelkirk** [comment]
Whoops - I inadvertently opened the PR with the old animation. I've pushed up the proper one now.

File: game/index.html (hunk 1 above)
Comment (dabreegster) [reply game/index.html]: This is some crazy CSS magic. Nice, I like it!

**Comment by dabreegster** [comment]
> I'm not sure where your osm demo lives, but if you like it, you could include it there too.
It's the same as `game`, with a few temporary hacks: https://github.com/dabreegster/abstreet/tree/mass_import_tmp
I'll deploy this there too. Thanks!

File: game/index.html (hunk 1 above)
Comment (michaelkirk) [reply game/index.html]: To be clear, I stole it wholesale (MIT License in-line).

**Pull request merged**